includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -1672,6 +1672,16 @@ private function parse_protocol( $url ) {
 		return null;
 	}
 
+	/**
+	 * Normalize a URL that appeared as a tag attribute.
+	 *
+	 * @param string $url The URL to normalize.
+	 * @return string The normalized URL.
+	 */
+	private function normalize_url_from_attribute_value( $url ) {
+		return preg_replace( '/\s/', '', $url );
```

Comment:
This isn't quite right. 

Only tabs and newlines should be stripped. See https://github.com/ampproject/amphtml/blob/af1e3a550feeafd732226202b8d1f26dcefefa18/validator/engine/parse-url.js#L217-L218

So this, along with the trimming, should be:

```suggestion
		return preg_replace( '/[\t\r\n]/', '', trim( $url ) );
```

Reply:
Sure, I'll make that change.

Reply:
It's applied in https://github.com/ampproject/amp-wp/pull/4166/commits/f19301917f553dd7cc66d593d2fa0e6a02160e49